inference-engine/tests/functional/plugin/cpu/single_layer_tests/reduce_ops.cpp

```diff
@@ -155,186 +269,421 @@ const std::vector<ngraph::helpers::ReductionType> reductionLogicalTypes = {
         ngraph::helpers::ReductionType::LogicalAnd
 };
 
-const std::vector<std::vector<size_t>> inputShapes = {
-        std::vector<size_t>{10, 5, 15, 12},
-        std::vector<size_t>{3, 5, 7, 9},
+std::vector<std::vector<ov::test::InputShape>> inputShapes = {
+    {{{}, {{2, 19, 2, 9}}}},
+    {{{{1, 5}, {2, 20}, {1, 5}, {1, 10}}, {{2, 9, 2, 2}, {2, 9, 2, 9}, {2, 19, 2, 9}}}},
+};
+
+std::vector<std::vector<ov::test::InputShape>> inputShapes_5D = {
+    {{{}, {{2, 19, 2, 2, 9}}}},
+    {{{{1, 5}, {2, 20}, {1, 5}, {1, 5}, {1, 5}}, {{2, 9, 2, 2, 2}, {2, 9, 3, 2, 2}, {2, 19, 3, 2, 2}}}},
+};
+
+std::vector<std::vector<ov::test::InputShape>> inputShapes_6D = {
+    {{{}, {{2, 19, 2, 2, 2, 2}}}},
+    {{{{1, 5}, {2, 20}, {1, 5}, {1, 5}, {1, 5}, {1, 5}}, {{2, 9, 2, 2, 2, 2}, {3, 9, 2, 2, 2, 2}, {3, 19, 2, 2, 2, 2}}}},
 };
 
 std::vector<CPUSpecificParams> cpuParams_4D = {
         CPUSpecificParams({nChw16c}, {nChw16c}, {}, {}),
-        CPUSpecificParams({nchw}, {nchw}, {}, {})
+        CPUSpecificParams({nchw}, {nchw}, {}, {}),
+        CPUSpecificParams({nhwc}, {nhwc}, {}, {})
 };
 
 std::vector<CPUSpecificParams> cpuParams_5D = {
         CPUSpecificParams({nCdhw16c}, {nCdhw16c}, {}, {}),
-        CPUSpecificParams({ncdhw}, {ncdhw}, {}, {})
+        CPUSpecificParams({ncdhw}, {ncdhw}, {}, {}),
+        CPUSpecificParams({ndhwc}, {ndhwc}, {}, {})
 };
 
-const auto paramsOneAxis = ::testing::Combine(
-        testing::Combine(
-            testing::ValuesIn(axes),
-            testing::ValuesIn(opTypes),
-            testing::ValuesIn(keepDims),
-            testing::ValuesIn(reductionTypes),
-            testing::ValuesIn(inpOutPrc),
-            testing::Values(InferenceEngine::Precision::UNSPECIFIED),
-            testing::Values(InferenceEngine::Precision::UNSPECIFIED),
-            testing::Values(InferenceEngine::Layout::ANY),
-            testing::ValuesIn(inputShapes),
-            testing::Values(CommonTestUtils::DEVICE_CPU)),
-        testing::Values(emptyCPUSpec));
-
-const auto paramsOneAxisLogical = testing::Combine(
+std::vector<CPUSpecificParams> cpuParams_HybridLayout_4D = {
+        CPUSpecificParams({nChw16c}, {}, {}, {}),
+        CPUSpecificParams({nhwc}, {}, {}, {})
+};
+
+std::vector<CPUSpecificParams> cpuParams_HybridLayout_5D = {
+        CPUSpecificParams({nCdhw16c}, {}, {}, {}),
+        CPUSpecificParams({ndhwc}, {}, {}, {})
+};
+
+const std::vector<fusingSpecificParams> fusingParamsSet {
+        /* activations */
+        fusingRelu,
+        fusingElu,
+        fusingTanh,
+        fusingSwish,
+
+        // TODO: Should be uncommented after fixing error "get_shape was called on a descriptor::Tensor with dynamic shape"
```

Comment:
Is it still actual? is yes can we add ticket number?

Reply:
Uncommented. Thanks Dmitry!